app/components/img-selector.js

```diff
@@ -15,6 +16,9 @@ export default Ember.Component.extend({
                 }
             };
             reader.readAsDataURL(e.target.files[0]);
+        },
+        clickInput: function() {
+            this.$().find('.img-selector-input').click();
```

Comment:
`this.$().find('.img-selector-input').click();` can be translated as:

"Find all the descendants of this specific component's element that match the class selector."
(https://api.jquery.com/find/)

 I will make the assumption that img-selector components are not being nested . That said I'd like to avoid the assumption that there will never ever be another .img-selector-input on the page, and this strategy helps restrict the subset of the DOM that might accidentally get 'clicked' (where as `Ember.$(...)` would select across the whole DOM).